src/rules/javascript/content-scripts-file-absent.js

```diff
@@ -0,0 +1,31 @@
+import { normalizePath } from 'utils';
+import { CONTENT_SCRIPT_NOT_FOUND } from 'messages/javascript';
+
+export default {
+  create(context) {
+    const existingFiles = context.settings.existingFiles || {};
+    return {
+      CallExpression(node) {
+        if (node.callee.type === 'MemberExpression' &&
+          node.callee.property &&
+          node.callee.property.name === 'executeScript') {
+          node.arguments.forEach((arg) => {
+            if (arg.type !== 'ObjectExpression') return;
+
```

Comment:
To make false positive less likely to happen because of this validation rule, we should also check if this executeScript call expression is happening on a actual WebExtension API object, e.g. like we do for the existent validation rule for the deprecated APIs:
https://github.com/mozilla/addons-linter/blob/6109428a065ff175e97ef483d1ee9a29ef3d94ad/src/rules/javascript/webextension-api.js#L9

Looking at the AST received by this CallExpression hook, it should be something like:
```js
isBrowserNamespace(node.callee.object.object.name);
```

We should also add some test cases that ensures that this rule doesn't generate the false positive errors, e.g. on JS fragments like:

```js
executeScript(...);

myObject.executeScript(...);

myObject.tabs.executeScript(...);
```